src/client/user/components/Drawer/ControlPanel/index.tsx

```diff
@@ -125,12 +153,18 @@ export default function ControlPanel() {
   const drawerStates = useDrawerState()
   const drawerIsOpen = drawerStates.controlPanelIsOpen
   const modalDispatch = useDrawerDispatch()
+  const dispatch = useDispatch()
 
   // Fetch short link state and dispatches from redux store through our helper hook.
   const { shortLinkState, shortLinkDispatch } = useShortLink(
     drawerStates.relevantShortLink!,
   )
 
+  // Toggle Link History
+  const isLinkHistoryToggled = drawerStates.linkHistoryIsToggled
```

Comment:
I feel this is a weak name, toggled doesn't tell whether the component is active or not, `isLinkHistoryActive` sounds better. But if `toggled` is commonly used in the project, feel free to resolve this comment.

Reply:
I agree it would be clearer